<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PIZZA</class>
 <widget class="QWidget" name="PIZZA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_teszta">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Milyen tésztával kéred?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Alap tészta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vékony tészta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vastag tészta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sajt szélű tészta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bacon szélű tészta</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_alap">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Milyen alappal kéred ?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Alap szósz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Paradicsomos alap</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tejszínes alap</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Csípős alap</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>20</width>
     <height>631</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_teszta_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Válassz feltétet</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kolbász     350,- Ft</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sonka        350,- Ft</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Szalámi     350,- Ft</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tojás          250,- Ft</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Articsóka          250,- Ft</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gomba      250,- Ft</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sajt                     250.- Ft</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Olajbogyó        250,- Ft</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Paprika              250, -Ft</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kukorica            250,- Ft</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_11">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>212</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Paradicsom      250,- Ft</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bacon        350,- Ft</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Sörök</string>
   </property>
   <widget class="QCheckBox" name="checkBox_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Kőbányai   590,- Ft</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_16">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Soproni       590,- Ft</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>250</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pepsi.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pepsi cola      390,- Ft</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>coca.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Coca cola   390,- Ft</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_hazhozszallitas">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>NÉV: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cím:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Telefon:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Megjegyzés</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Futár rendelés</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>wolt.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>441</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>451</width>
       <height>581</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>241</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <column>
       <property name="text">
        <string>Új oszlop</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ár</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_total_price">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>310</y>
        <width>221</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ÖSSZESEN :  </string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <widget class="QRadioButton" name="radioButton_hazhozszallitas">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>191</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Házhozszállítás   590,- Ft</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_Helyben">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>171</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Helyben fogyasztás</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pizzeria Napoletana</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1102 Budapest, Kőbányai út 30</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>301</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Házhoz szállított étel - NYUGTA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_total_price_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>401</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fizetendő  összesen:  </string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>291</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kiszállítással megbízott :  Wolt  125- futár</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Megrendelő neve:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>261</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Címe:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>351</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Telefon:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>230</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>telefon</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Megjegyzés:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>250</y>
       <width>311</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>A futárnak</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>label_teszta</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_alap</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>label_teszta_2</zorder>
  <zorder>checkBox</zorder>
  <zorder>checkBox_2</zorder>
  <zorder>checkBox_3</zorder>
  <zorder>checkBox_4</zorder>
  <zorder>checkBox_5</zorder>
  <zorder>checkBox_6</zorder>
  <zorder>checkBox_7</zorder>
  <zorder>checkBox_8</zorder>
  <zorder>checkBox_9</zorder>
  <zorder>checkBox_10</zorder>
  <zorder>checkBox_11</zorder>
  <zorder>checkBox_12</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>line_3</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>frame_hazhozszallitas</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
